### PR TITLE
variangs/novacustom_v5.0tu: add v0.9.0 release notes

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

File: docs/variants/novacustom_v560tu/releases.md [hunk 1]
```diff
@@ -6,7 +6,78 @@ NovaCustom V560TU
...
+[sha256][novacustom_v56x_mtl_v0.9.0.rom_hash]{.md-button}
+[sha256.sig][novacustom_v56x_mtl_v0.9.0.rom_sig]{.md-button}
+
+[novacustom_v56x_mtl_v0.9.0_dev_signed.rom][novacustom_v56x_mtl_v0.9.0_dev_signed.rom_file]{.md-button}
+[sha256][novacustom_v56x_mtl_v0.9.0_dev_signed.rom_hash]{.md-button}
+[sha256.sig][novacustom_v56x_mtl_v0.9.0_dev_signed.rom_sig]{.md-button}
+
+To verify binary integrity with hash and signature please follow the
+instructions in [Dasharo release signature verification](/guides/signature-verification)
+using [this key](https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/customer-keys/novacustom/novacustom-open-source-firmware-release-0.x-key.asc)
```
Comment (BeataZdunczyk) [inline docs/variants/novacustom_v560tu/releases.md:67]: 404

File: docs/variants/novacustom_v540tu/releases.md [hunk 2]
```diff
@@ -6,7 +6,78 @@ NovaCustom V540TU
...
+[sha256][novacustom_v54x_mtl_v0.9.0.rom_hash]{.md-button}
+[sha256.sig][novacustom_v54x_mtl_v0.9.0.rom_sig]{.md-button}
+
+[novacustom_v54x_mtl_v0.9.0_dev_signed.rom][novacustom_v54x_mtl_v0.9.0_dev_signed.rom_file]{.md-button}
+[sha256][novacustom_v54x_mtl_v0.9.0_dev_signed.rom_hash]{.md-button}
+[sha256.sig][novacustom_v54x_mtl_v0.9.0_dev_signed.rom_sig]{.md-button}
+
+To verify binary integrity with hash and signature please follow the
+instructions in [Dasharo release signature verification](/guides/signature-verification)
+using [this key](https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/customer-keys/novacustom/novacustom-open-source-firmware-release-0.x-key.asc)
```
Comment (BeataZdunczyk) [inline docs/variants/novacustom_v540tu/releases.md:67]: 404

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]
Besides two 404 errors, LGTM

File: docs/variants/novacustom_v540tu/releases.md (hunk 2 above)
Comment (mkopec) [reply docs/variants/novacustom_v540tu/releases.md]: fixed in https://github.com/Dasharo/docs/pull/858/commits/ad1a62a1572a679c9b641aae31425891687676fb

File: docs/variants/novacustom_v560tu/releases.md (hunk 1 above)
Comment (mkopec) [reply docs/variants/novacustom_v560tu/releases.md]: fixed in https://github.com/Dasharo/docs/pull/858/commits/ad1a62a1572a679c9b641aae31425891687676fb

**Review by BeataZdunczyk — APPROVED** [review]

**Pull request merged**